code/modules/mapping/mapping_helpers.dm

```diff
@@ -257,118 +257,126 @@
 		log_mapping("[src] spawned outside of mapload!")
 		return
 
-	var/obj/machinery/power/apc/apc = locate(/obj/machinery/power/apc) in loc
-	if(!apc)
+	var/obj/machinery/power/apc/target = locate(/obj/machinery/power/apc) in loc
+	if(isnull(!target))
 		log_mapping("[src] failed to find an apc at [AREACOORD(src)]")
 	else
-		payload(apc)
+		payload(target)
 
 /obj/effect/mapping_helpers/apc/LateInitialize()
 	. = ..()
-	var/obj/machinery/power/apc/apc = locate(/obj/machinery/power/apc) in loc
+	var/obj/machinery/power/apc/target = locate(/obj/machinery/power/apc) in loc
 
-	if(!apc)
+	if(isnull(target))
 		qdel(src)
 		return
-	if(apc.cuttedAIwire)
-		apc.wires.cut(WIRE_AI)
-	if(apc.cell_5k)
-		apc.install_cell_5k()
-	if(apc.cell_10k)
-		apc.install_cell_10k()
-	if(apc.unlocked)
-		apc.unlock()
-	if(apc.syndicate_access)
-		apc.give_syndicate_access()
-	if(apc.away_general_access)
-		apc.give_away_general_access()
-	if(apc.no_charge)
-		apc.set_no_charge()
-	if(apc.full_charge)
-		apc.set_full_charge()
-	if(apc.cell_5k & apc.cell_10k)
-		CRASH("Non-combinable apc helpers!")
-	if(apc.syndicate_access & apc.away_general_access)
-		CRASH("Non-combinable apc helpers!")
-	if(apc.no_charge & apc.full_charge)
-		CRASH("Non-combinable apc helpers!")
-	apc.update_appearance()
+	if(target.cutted_AI_wire)
+		target.wires.cut(WIRE_AI)
+	if(target.cell_5k)
+		target.install_cell_5k()
+	if(target.cell_10k)
+		target.install_cell_10k()
+	if(target.unlocked)
+		target.unlock()
+	if(target.syndicate_access)
+		target.give_syndicate_access()
+	if(target.away_general_access)
+		target.give_away_general_access()
+	if(target.no_charge)
+		target.set_no_charge()
+	if(target.full_charge)
+		target.set_full_charge()
+	if(target.cell_5k && target.cell_10k)
+		CRASH("Tried to combine non-combinale cell_5k and cell_10k apc helpers!")
+	if(target.syndicate_access && target.away_general_access)
+		CRASH("Tried to combine non-combinale syndicate_access and away_general_access apc helpers!")
+	if(target.no_charge && target.full_charge)
+		CRASH("Tried to combine non-combinale no_charge and full_charge apc helpers!")
```

Comment:
it's `combinable`, you're missing the second `b`. should also capitalize APC in these crash messages since it's an acronym